src/dsverifier.cpp

```diff
@@ -2243,8 +2720,8 @@ void check_filter_phase_det(void)
   int freq_response_samples = 100;
   double w;
   double w_incr = 1.0 / freq_response_samples;
-  double res[freq_response_samples + 1];
-  double _res[freq_response_samples + 1];
+  double res[101];
```

Comment:
What is the meaning of `101`? It seems to be a magic number.

Reply:
I don't know about that and I'm not sure who developed that code. I just changed because cpplint script was complaining about using "var = 100; res[var+1]..." and asked to change to "res[101]...".

Reply:
I would suggest to keep `[freq_response_samples + 1]`.